website/docs/r/api_management.html.markdown

```diff
@@ -357,6 +361,8 @@ In addition to all arguments above, the following attributes are exported:
 
 * `tenant_access` - The `tenant_access` block as documented below.
 
+* `certificate_information` - The `certificate_information` block as documented below.
```

Comment:
is information really needed here?
```suggestion
* `certificate` - The `certificate_information` block as documented below.
```

Reply:
But there is a block `certificate` here https://github.com/terraform-providers/terraform-provider-azurerm/pull/12262/files#diff-8b062899963fc25bde31a53d5291f66e189ed72c2a8f2d33215b231ccc27c374R225

Reply:
what is this block then? differences between the two?

Reply:
It's like the `certificate` is a block to set the certificate type and `certificate_information` is  the computed certificate information.
![image](https://user-images.githubusercontent.com/56525716/128116007-bc5dcf04-3d18-4cb2-a531-e6dc886936d1.png)

Reply:
that `certificate` block is not in the policy block? so we can drop information?

![image](https://user-images.githubusercontent.com/1638467/130530224-6c63cc5a-c290-4340-a4f4-96cc1677e485.png)